### PR TITLE
doc(email-connector): create a documentation for the new email connector

### PR DESCRIPTION
**Pull request by mathias-vandaele**
…tor (inbound and outbound)

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [x] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:87]: comment for the connector ifself, can we return `sizeInBytes` instead of `size`?

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:546]: I would highlight this, maybe a warn/note component, you can find examples in other connectors docs

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:558]: We don't describe any params here, is it expected?

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:565]: Let us clarify this part, as this is super important. 
It's unclear to me, so I imagine it can be the same for a customer.

I don't have a good description in mind, but we can discuss

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:572]: this part is great!

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:605]: > Friendly Enough Expression Language (FEEL)

This could be a link to the docs

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:615]: the rendering is weird there
Screenshot 2024-09-17 at 10 10 57

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:618]: > Consume unmatched
This could be a link I guess we have it documented somewhere

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (johnBgood) [inline docs/components/connectors/out-of-the-box-connectors/email.md:633]: do we want to support Start Events ?

**Review by johnBgood — COMMENTED** [review]
Few comments only, overall great 🔥

**Comment by christinaausley** [comment]
@mathias-vandaele If you run `npm run format` in your terminal and commit the changes, the Prettier linting will clean up these errors for you 👍

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (mathias-vandaele) [reply docs/components/connectors/out-of-the-box-connectors/email.md:626]: No I will remove this

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (mathias-vandaele) [reply docs/components/connectors/out-of-the-box-connectors/email.md:2]: I am trying to understand why

**Comment by mesellings** [comment]
@mathias-vandaele I can do a TW review for you 👍

**Comment by mesellings** [comment]
@mathias-vandaele Can I commit directly to the branch, or are you still working on it? I can fix those issues for you at the same time as well if you like (the sidebar etc)

**Comment by mathias-vandaele** [comment]
> @mathias-vandaele Can I commit directly to the branch, or are you still working on it? I can fix those issues for you at the same time as well if you like (the sidebar etc)

Hello @mesellings, Yes I am done with it, and if you could fix those issues for me, that would be amazing, thank you 😊

File: docs/components/connectors/out-of-the-box-connectors/email.md
Comment (mesellings) [reply docs/components/connectors/out-of-the-box-connectors/email.md:2]: Fixed for you - it needed an entry in the sidebars.js file.

**Review by mesellings — DISMISSED** [review]
@mathias-vandaele I have committed quite a lot of TW edits as follows:

* Fixed the sidebar issue
* Rewording for grammar, readability, standards, active voice etc
* General reformatting as above

Happy to approve, great job 👍

**Comment by mesellings** [comment]
@mathias-vandaele did you want me to merge this, and when did youw ant the docs released - can they go out now in Next for example?

**Comment by mathias-vandaele** [comment]
It will probably be release on 8.6 if testing goes well @mesellings 

I still don't see the email tab on the sidebars on the preview deployment, do you know why?

**Comment by mesellings** [comment]
@mathias-vandaele Do you not see it here?

image

**Comment by mathias-vandaele** [comment]
> @mathias-vandaele Do you not see it here?
> 
> image

Thanks a lot, I see that it is in `/next` that's why i was not seeing it!

**Comment by mesellings** [comment]
> > @mathias-vandaele Do you not see it here?
> > image
> 
> Thanks a lot, I see that it is in `/next` that's why i was not seeing it!

No problem - will this need to be backported for earlier versions as well, or is the new connector only for 8.6?

**Comment by mathias-vandaele** [comment]
> > > @mathias-vandaele Do you not see it here?
> > > image
> > 
> > 
> > Thanks a lot, I see that it is in `/next` that's why i was not seeing it!
> 
> No problem - will this need to be backported for earlier versions as well, or is the new connector only for 8.6?

only 8.6 👍

**Comment by akeller** [comment]
Is the `hold` label here just for the 8.6 release? If so, `/next/` is our 8.6 preview and ok to merge into. If you wait until we cut the release, you'll need to backport your changes to the new `/next/` which will be 8.7.

**Comment by mesellings** [comment]
@akeller I was just going to wait until closer (next week) to see if this actually goes into the 8.6 release before I merge the docs in, as it sounds like it is not 100% certain to go in until testing is complete:

> It will probably be release on 8.6 if testing goes well

Or do you think we should merge it in?

**Comment by akeller** [comment]
> @akeller I was just going to wait until closer (next week) to see if this actually goes into the 8.6 release before I merge the docs in, as it sounds like it is not 100% certain to go in until testing is complete:
> 
> > It will probably be release on 8.6 if testing goes well
> 
> Or do you think we should merge it in?

@mesellings Ah, thanks for the info - definitely makes sense to wait then.

**Review by johnBgood — APPROVED** [review]

**Comment by mesellings** [comment]
@johnBgood I see you have removed the inbound connector - is this because that will not be included in the 8.6 release?

**Comment by mesellings** [comment]
@mathias-vandaele Is this okay to merge, or is the email connector not 100% in the 8.6 release yet? 

Also, I see @johnBgood  removed the inbound connector docs - is this because that will not be included in the 8.6 release?

**Comment by johnBgood** [comment]
@mesellings I didn't commit to this PR :o But we decided to remove the Inbound part of this connector indeed. 
I'll let @mathias-vandaele elaborate on that  :)

**Comment by mesellings** [comment]
> @mesellings I didn't commit to this PR :o But we decided to remove the Inbound part of this connector indeed. I'll let @mathias-vandaele elaborate on that :)

Thanks @johnBgood my apologies, thought it was your commits 😊

**Pull request merged**
